pkg/install/v1alpha1/jiva_volume_0.7.0.go

```diff
@@ -571,6 +571,8 @@ spec:
   post: |
     {{- $replicaAntiAffinity := jsonpath .JsonResult "{.metadata.labels.openebs\\.io/replica-anti-affinity}" | default "none" -}}
     {{- trim $replicaAntiAffinity | saveAs "creategetpvc.replicaAntiAffinity" .TaskResult | noop -}}
+    {{- $targetAffinity := jsonpath .JsonResult "{.metadata.labels.openebs\\.io/target-affinity}" | default "none" -}}
```

Comment:
Let us have the `trim` here itself just before `default` is invoked

Reply:
done